src/main/java/javaslang/collection/CharSeq.java

```diff
@@ -99,7 +99,7 @@ public static CharSeq of(char... characters) {
         Objects.requireNonNull(characters, "characters is null");
         final char[] chrs = new char[characters.length];
         System.arraycopy(characters, 0, chrs, 0, characters.length);
-        return new CharSeq(new String(chrs));
+        return characters.length == 0 ? empty() : new CharSeq(new String(chrs));
```

Comment:
Hi Simone!
I would made this check before 
```
final char[] chrs = ...
```
to avoid instantiate empty char[]